src/main/scala/org/allenai/plugins/ScaladocGenPlugin.scala

```diff
@@ -216,7 +216,10 @@ object ScaladocGenPlugin extends AutoPlugin {
         scalaApiMappingsSetting,
         // This will pull in links from pom.xml files that sbt has generated, when the project sets
         // the apiURL key.
-        autoAPIMappings := true
+        autoAPIMappings := true,
+        // This adds the output of Unidoc to the SbtSite plugin's site directory (what gets synced
+        // to github pages).
```

Comment:
`sbt-site` just copies files into `target/site`, based on mappings. It's like the universal plugin's `stage` command.

`sbt-ghpages` then takes this directory and copies it to the `gh-pages` branch, then pushes it to github.

This is saying "take the mappings unidoc provides for the site plugin, and put the output in "latest/api"".